pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -332,6 +335,90 @@ func TestAccLocalNugetRepository(t *testing.T) {
 	})
 }
 
+var commonJavaParams = map[string]interface{}{
+	"name":                            "",
+	"checksum_policy_type":            "client-checksums",
+	"snapshot_version_behavior":       "unique",
+	"max_unique_snapshots":            randSelect(0, 5, 10),
+	"handle_releases":                 true,
+	"handle_snapshots":                true,
+	"suppress_pom_consistency_checks": false,
+}
+
+const localJavaRepositoryBasic = `
+		resource "{{ .resource_name }}" "{{ .name }}" {
```

Comment:
I thought the convention for indentation is a tab character? @danielmkn have you run `go fmt` on this?

Reply:
yes, but gofmt won't change the string in const. This is just the template string

Reply:
and it looks the same as the actual .tf file after applying 'terraform fmt', so we are good there